packages/layouts-css/src/page.scss

```diff
@@ -94,5 +96,71 @@
         justify-content: flex-end;
       }
     }
+
+    &-centered {
+      max-width: 1440px;
+      width: 100%;
+    }
+
+    &-title-area {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: calc(var(--iui-size-s) * 3);
+    }
   }
+
+  &-bottom-bar {
+    grid-area: bottom-bar;
+    height: calc(var(--iui-size-s)*5 + var(--iui-size-l)); //this is the sizes of the two bottom buttons of side nav
+  }
+
+  &-action-bar {
+    background-color: var(--iui-color-background);
+    border-top: 2px solid var(--iui-color-border);
+    margin-inline: auto;
+    padding: var(--iui-size-m);
+    height: inherit;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-areas: 'left center right';
+
+    //#region Add env(safe-area-inset-*) for iPhone notch on the left & right sides of content area
+
+    @include small-monitor {
+      padding-inline: unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-right))') unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-left))');
+    }
+
+    @include tablet {
+      padding-inline: unquote('max(var(--iui-size-xl), env(safe-area-inset-right))') unquote('max(var(--iui-size-xl), env(safe-area-inset-left))');
```

Comment:
Maybe switch template to vertical display to make it fit into page.
You might also need to adjust initial height and justify-content settings in areas.

Reply:
I don't really understand what you mean by switching template to vertical display. Could you elaborate more? Do you mean for only `tablet` case?

Reply:
You could add something like this to mobile version of action bar: 
![image](https://user-images.githubusercontent.com/83585998/235885037-62b279f8-815b-4573-9b13-f0a7a889643a.png)
![image](https://user-images.githubusercontent.com/83585998/235885537-52a1b899-0c91-4041-b009-93a8e616f868.png)

It would make grid to display items in one column:
![image](https://user-images.githubusercontent.com/83585998/235885683-438fa5da-091a-427b-a842-1303e60ff11b.png)

@FlyersPh9 if that is ok?

Reply:
Thanks for the guide. I have updated this. Only thing would be that it no longer aligns with the side nav buttons but I think ultimately the specifics of content will be up to the users to adjust accordingly.

Reply:
I'm fine with this for now.  

We don't have a great solution for mobile at the moment.  A while back we explored replacing the side navigation with a bottom bar in #34, if we ever did implement that the double bottom bar will be kind of strange.  But for where we are now, this suggestion makes sense to me.